test/Interop/Cxx/stdlib/print-swiftconcurrencyshims-interface.swift

```diff
@@ -1,10 +0,0 @@
-// RUN: %target-swift-ide-test -print-module -module-to-print=_SwiftConcurrencyShims -source-filename=x -enable-experimental-cxx-interop | %FileCheck %s
```

Comment:
@egorzhdan can you review this change. Is there some reason we'd want to import this function? It conflicts with `Darwin.exit`.

Reply:
I think it used from `swift/stdlib/public/Concurrency/Task.swift`, I wonder if `_Concurrency` still builds if we don't import this function.

Reply:
I think re-declaring `exit()` in the shim header isn't really great, it would be nice to just import `Darwin.exit`/`Glibc.exit`/etc from `Task.swift` and use that directly from Swift.

Reply:
I'm fine with either solution as long as _Concurrency builds with interop enabled 🙂

Reply:
Mind taking another look?